cypress/e2e/po/components/sortable-table.po.ts

```diff
@@ -56,6 +56,14 @@ export default class SortableTablePo extends ComponentPo {
     return cy.getId('sortable-table-promptRemove');
   }
 
+  downloadYamlButton() {
+    return cy.getId('sortable-table-download');
+  }
+
+  downloadKubeConfigButton() {
```

Comment:
this is specific to the prov cluster table and should be in that po.

slightly longer, but could also use rowActionMenuOpen(row name).getMenuItem('Download KubeConfig')